styleguide/_themes/derek/scss/snowflakes/homepage.scss

```diff
@@ -302,136 +390,3 @@
     color: $brand-primary;
   }
 }
-
-
-//solutions
-
-.solutions-hoverArea {
-  text-decoration: none;
-
-  &:hover {
-
-    .border-white {
-      border-color: $mustard;
-    }
-
-    h6,
-    .rsweb {
-      color: $mustard;
-    }
-  }
-}
-
-
-//quadrant
-
-.quadrant {
-  background-image: url('../imgs/derek/quadrant.jpg');
-  background-position: 23% bottom;
-  background-repeat: no-repeat;
-  background-size: 35%;
-  padding: 20px 0;
-  padding-bottom: 0;
-}
-
-.quadrant-row {
-  @include make-row;
-}
-
-.quadrant-textContainer {
-  @include make-sm-column(6);
-  @include make-sm-column-push(6);
-}
-
-.quadrant-title,
-.quadrant-text {
-  text-align: right;
-}
-
-.quadrant-buttonContainer {
-  @include make-sm-column(6);
-  @include make-sm-column-push(6);
-  @include make-xs-column(6);
-  @include make-xs-column-push(3);
-  padding: $half-padding-full;
-  text-align: right;
-}
-
-.logoShowcase-image {
```

Comment:
I lied. Also logo showcase which has also been off the site for basically a year.